pkg/controller/v1alpha2/experiment/experiment_controller.go

```diff
@@ -29,6 +30,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	experimentsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/experiment/v1alpha2"
+	trialsv1alpha2 "github.com/kubeflow/katib/pkg/api/operators/apis/trial/v1alpha2"
+	"github.com/kubeflow/katib/pkg/controller/v1alpha2/experiment/util"
 )
 
 var log = logf.Log.WithName("controller")
```

Comment:
I think, we should name this controller `experiment-controller`, as @gaocegege noticed.